docs/zz_generated.seed.ce.yaml

```diff
@@ -79,7 +79,24 @@ spec:
           # found at https://azure.microsoft.com/en-us/global-infrastructure/locations/
           location: ""
         # Baremetal contains settings for baremetal clusters in datacenters.
-        baremetal: {}
+        baremetal:
+          tinkerbell:
+            # Images represents standard VM Image sources.
+            images:
+              # HTTP represents a http source.
+              http:
+                # OperatingSystems represents list of supported operating-systems with their URLs.
+                operatingSystems:
+                  centos:
+                    <<version>>: <<url>>
```

Comment:
Instead of a placeholder like this, could we use an example string, like `vX.Y: http://example.com/images/centos.iso`? That would communicate the format better.